app/src/ui/app.tsx

```diff
@@ -535,10 +535,22 @@ export class App extends React.Component<IAppProps, IAppState> {
   }
 
   public componentDidMount() {
-    document.ondragover = document.ondrop = e => e.preventDefault()
+    document.ondragover = e => {
+      if (this.isShowingModal != null) {
```

Comment:
Oh! I totally did not realize that isShowingModal wasn't returning a boolean. That's a bug as far as I'm concerned and while I do feel a little bit bad about piling on to your PR I would really appreciate if you could help us fix that. If you change the `isShowingModal` implementation to something like this:

```ts
  private get isShowingModal() {
    return this.state.currentPopup !== null || this.state.errors.length > 0
  }
```

Then we should be able to change this line to be just `if(this.isShowingModal)` which would be much nicer in my opinion.

Reply:
Indeed that's a much cleaner solution 👍